modules/context/repo.go

```diff
@@ -100,6 +102,23 @@ func (r *Repository) CanUseTimetracker(issue *models.Issue, user *models.User) b
 		r.IsWriter() || issue.IsPoster(user.ID) || issue.AssigneeID == user.ID)
 }
 
+// GetCommitsCount returns cached commit count for current view
+func (r *Repository) GetCommitsCount() (int64, error) {
+	if r.IsViewBranch {
+		return cache.GetInt64(fmt.Sprintf("branch-commits-count-%d-%s", r.Repository.ID, r.BranchName), func() (int64, error) {
```

Comment:
Could we add helper functions for these cache keys instead of having "magic" strings? e.g
```
func BranchCommitsCountKey(repoID int64, branchName string) string { ... }
```
With helper functions, we don't have to worry about mistyped strings, and it makes the code cleaner IMO